helix-core/src/test/java/org/apache/helix/integration/task/TestJobFailure.java

```diff
@@ -118,8 +118,15 @@ public void testNormalJobFailure(String comment, List<String> taskStates,
   }
 
   private Map<String, Map<String, String>> createPartitionConfig(List<String> taskStates,
-      List<String> expectedTaskEndingStates) {
+      List<String> expectedTaskEndingStates) throws Exception {
     Map<String, Map<String, String>> targetPartitionConfigs = new HashMap<>();
+    // Make sure external view is has been created to the resource
```

Comment:
Make sure external view has been created for the resource...

Reply:
Done.